ci-operator/config/redhat-operator-ecosystem/OWNERS

```diff
@@ -0,0 +1,5 @@
+approvers:
```

Comment:
Neat idea putting the `OWNERS` into the org dir but unfortunately it looks like the check is not happy anyway. You'll either need to put `OWNERS` to the `playground` directory (in both `config` and `jobs`) or improve the [check](https://github.com/openshift/release/blob/master/hack/validate-owners.sh) to allow having just org-level `OWNERS`

Reply:
Moved down in the hierarchy for now.